### DISCUSSION TITLE
Main Monitor is taller

### DISCUSSION THREAD

**Discussion by hjrobins** [valinet/ExplorerPatcher#3522] [General]
When Monitor 1 is main
![image](https://github.com/user-attachments/assets/96d78cca-7524-4092-b066-e720dadab536)

When Monitor 2 is main
![image](https://github.com/user-attachments/assets/d1c6d5db-5857-434b-b209-25111f5b4983)

It doesn't matter if the taskbar icon size is Small or Large. I prefer it being small.
Windows 10 style on Windows 11.

**Comment by Amrsatrio** [comment]
Can't you resize the taskbar? Right click the taskbar > Unlock all taskbars > Drag the top border of the taskbar.

**Comment by hjrobins** [comment]
Ah yes that worked, thank you.
________________________________
